aptos-move/mvhashmap/src/versioned_data.rs

```diff
@@ -266,48 +439,121 @@ impl<K: Hash + Clone + Debug + Eq, V: TransactionWrite> VersionedData<K, V> {
         );
     }
 
+    /// Delete an entry from transaction 'txn_idx' at access path 'key' for BlockSTMv2.
+    /// Returns read dependencies from the entry that are no longer valid, panics if
+    /// the entry does not exist.
+    pub fn remove_v2<const ONLY_COMPARE_METADATA: bool>(
```

Comment:
Not yet connected?

Reply:
It's coming (drumbeats) https://github.com/aptos-labs/aptos-core/pull/16893